src/main/webapp/WEB-INF/tags/module/purap/camsDetail.tag

```diff
@@ -234,8 +234,7 @@
                 <purap:camsLocation camsLocationAttributes="${camsLocationAttributes}" ctr="${ctr}"
                                     ctr2="new"
                                     camsAssetLocationProperty="${locationPrefix}newPurchasingCapitalAssetLocationLine"
-                                    availability="${availability}" poItemInactive="${poItemInactive}"
-                                    fullEntryMode="${fullEntryMode}"/>
+                                    availability="${availability}" poItemInactive="${poItemInactive}"/>

```

Comment:
It was something I had missed when making changes earlier and didn't notice until I tried testing this one by opening a REQS via the webapp. I could separate it out, but it seemed easier to just include the fix here.